src/native/thrift/nativeConnection.ts

```diff
@@ -21,15 +22,19 @@ import { WebviewHandler } from 'native/webviewApi';
 declare global {
     interface Window {
         nativeConnections: { [id: string]: NativeConnection };
-        AndroidWebViewMessage?: (nativeMessage: NativeMessage) => {};
+        AndroidWebViewRequest?: (nativeMessage: NativeMessage) => {};
```

Comment:
What do you think about changing these names?
`AndroidWebViewRequest?`
`AndroidNativeResponse?`

`iOSWebViewRequest`
`iOSNativeResponse`

`receiveNativeRequest`